### DISCUSSION TITLE
Cannot put uploaded dag into the 'active state'.  I can for a local docker-compose version.

### DISCUSSION THREAD

**Discussion by bruciebruce** [apache/airflow#19266] [General]
### Official Helm Chart version

1.2.0 (latest released)

### Apache Airflow version

2.1.4 (latest released)

### Kubernetes Version

Client Version: version.Info{Major:"1", Minor:"21+", GitVersion:"v1.21.2-13+d2965f0db10712", GitCommit:"d2965f0db1071203c6f5bc662c2827c71fc8b20d", GitTreeState:"clean", BuildDate:"2021-06-26T01:02:11Z", GoVersion:"go1.16.5", Compiler:"gc", Platform:"linux/amd64"} Server Version: version.Info{Major:"1", Minor:"18+", GitVersion:"v1.18.20-eks-8c579e", GitCommit:"8c579edfc914f013ff48b2a2b2c1308fdcacc53f", GitTreeState:"clean", BuildDate:"2021-07-31T01:34:13Z", GoVersion:"go1.13.15", Compiler:"gc", Platform:"linux/amd64"} WARNING: version difference between client (1.21) and server (1.18) exceeds the supported minor version skew of +/-1

### Helm Chart configuration

I am using the default helm chart created by the airflow group.

### Docker Image customisations

None.

### What happened

I installed airflow with the helm chart from the airflow people:

https://airflow.apache.org/docs/helm-chart/stable/index.html

specifically these 3 steps:

kubectl create namespace airflow
helm repo add apache-airflow https://airflow.apache.org
helm install airflow apache-airflow/airflow --namespace airflow

to a k8s 3 node cluster hosted on AWS.

I run airflow locally provided by the docker-compose.yaml file provided here:

https://airflow.apache.org/docs/apache-airflow/stable/start/docker.html

The local docker verson runs fine without any headaches.

However when I deploy the SAME dag that I deploy locally (which works just fine) to my cluster with this command:

kubectl cp hello-world.py airflow/airflow-scheduler-xxxxxxxxxx-xxxx:/opt/airflow/dags -c scheduler

the dag is stuck in a paused state.  I cannot put the dag in an active state by clicking in the UI.

### What you expected to happen

To put the dag in an active state.

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Comment by potiuk** [comment]
You probably don't understand that helm chart is "production" setup. You just don't copy files to container via kubectl like that. When you use helm chart you need to choose which is the dag synchronisationt that should be used - whether to use DAGs baked-in, GitSync/ PVC and then you need to make sure that this file is synchronized using one of those mechanisms.

See https://airflow.apache.org/docs/helm-chart/stable/manage-dags-files.html

**Reply by bruciebruce** [reply]
Thank you for getting back.  I will investigate this.
